tools/trap/ramdumpParser.py

```diff
@@ -820,6 +820,34 @@ def main():
 				ext = (line.split("=")[1].strip())
 	elf = (BIN_PATH + 'tinyara' + ext)
 
+	if not log_file and not dump_file:
+		print('Usage error: Must specify one of the -t or -e options. Plz find below for proper usage')
+		usage()
+
+	access_check_list = [
+		("gdb", gdb_path)
+		("nm", nm_path),
+		("readelf", readelf_path)
+	]
+
+	exist_check_list = [
+		("dump_file", dump_file),
+		("log_file", log_file),
+		("elf", elf)
+	] + access_check_list
+
+	for name, path in exist_check_list:
+		if path and not os.path.exists(path):
+			print(f"{path} does not exist. Please provide the proper path for {name}...")
+			sys.exit(1)
+
+	for name, path in access_check_list:
+		if not os.path.exists(readelf_path):
```

Comment:
There is a missing about checking permission.
Please check this.

Reply:
Fixed it.